ldb/management/commands/match_csa.py

```diff
@@ -0,0 +1,77 @@
+import datetime
+
+import reversion
+from django.db import transaction
+from xlrd import open_workbook
+from django.core.management import BaseCommand
+
+from ldb.models import Student, Member
+
+
+def cell(sheet, row, col):
+    value = sheet.cell(row, col).value
+    if value and value != '':
+        return value
+    else:
+        return None
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument('file', nargs=1, type=unicode)
+
+        parser.add_argument('--yes-value',
+                            dest='yes-value',
+                            default='ja',
+                            help='Define which value is in the document when the CSa says the person is still a student')
+
+    def check_is_student(self, studenten, row, yes_value):
+        value = cell(studenten, row, 1)
+        return value.lower() == yes_value.lower()
+
+    def get_student_number(self, studenten, row):
+        return str(int(cell(studenten, row, 0)))
+
+    def handle(self, *args, **options):
+        book = open_workbook(options['file'])
```

Comment:
Omgeschreven naar gebruik van csv. Misschien dat we nog even moeten kijken naar delimiters enzo, maar heb nu gewoon standaard [reader](https://docs.python.org/2/library/csv.html#csv.reader) gepakt.